index.html

```diff
@@ -0,0 +1,180 @@
+<DOCTYPE! html>
+<html>  
+  <head>
+    <!-- Latest compiled and minified CSS -->
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap.min.css">
+    <link rel="stylesheet" href="styles.css">    
+  </head>
+  <title>Welcome to Danebook</title>
+  <body>
+    <div class="container">
+      <nav class="row">
+        <div class="col-xs-7">

```

Comment:
Every column is sized "xs" and it makes the page look squished and unreadable at narrow screens. "md" might be more appropriate.